lib/debugger-wrapper.js

```diff
@@ -116,23 +120,26 @@ function handleDebuggerPausedEvent(message) {
     filterName: null,
     // TODO: moduleUtils does quite a lot of work to split this up;
     // TODO: maybe it shouldn't?
-    sourceUri:`file://${bpData.moduleInfo.baseDir}/` +
-      bpData.moduleInfo.scriptRelativePath,
+    sourceUri:`file://${bpData.moduleInfo.baseDir}/${bpData.moduleInfo.scriptRelativePath}`,
     sourceCrc32c: null,
     breakpointId,
   };
 
   transmitter.addEvent({methodEntry});
 }
 
-function normalizeScriptPath(scriptPath) {
-  // Remove file prefix which was added in Node v10.12
-  if (scriptPath.startsWith('file://') && path.sep === '/') {
-    scriptPath = scriptPath.substring('file://'.length);
-  } else if (scriptPath.startsWith('file:///') && path.sep === '\\') {
-    scriptPath = scriptPath.substring('file:///'.length).replace(/\//g, '\\');
+function ignorePause(pauseContext) {
+  if (pauseContext.reason !== 'other') {
+    debug(`ignoring debugger pause due to reason being ${pauseContext.reason}.`);
+    return true;
   }
-  return scriptPath;
+
+  if (pauseContext.hitBreakpoints.length === 0) {
```

Comment:
Is this field definitely present when empty?

Reply:
no, it could be undefined! adding to the test